docs/Tutorial_en.md

```diff
@@ -235,3 +235,105 @@ fmt.Println(myFact.WhatToSay)
 // this should prints
 // Lets Say "Hello Grule"
 ```
+
```

Comment:
I think, this documentation portion is too advanced for those who just start learning how to use Grule. 
I would advise you to move this portion to a separate .md file and have it linked from `FAQ_en.md` under a new FAQ entry. eg. Q : How can I test which of rules I define is valid for a given Facts... A : You can use engine.FetchMatchingRule function as [documented here](link). Something like that... :)